rules/sonic-thermalctld.mk

```diff
@@ -2,5 +2,5 @@
 
 SONIC_THERMALCTLD = python-sonic-thermalctld_1.0-1_all.deb
 $(SONIC_THERMALCTLD)_SRC_PATH = $(SRC_PATH)/sonic-platform-daemons/sonic-thermalctld
-$(SONIC_THERMALCTLD)_WHEEL_DEPENDS = $(SONIC_DAEMON_BASE_PY2)
+$(SONIC_THERMALCTLD)_WHEEL_DEPENDS = $(SONIC_PY_COMMON_PY2) $(SONIC_DAEMON_BASE_PY2)
```

Comment:
>$(SONIC_PY_COMMON_PY2) $(SONIC_DAEMON_BASE_PY2) [](start = 37, length = 47)

I see some code duplication in both repo. Can we merge 2 into 1?

Reply:
That's the ultimate goal. After this merges, I intend to shift all code away from sonic-daemon-base and sonic-device-util, then remove them. Please read the "Next Steps" in my description above and see the linked issue: https://github.com/Azure/sonic-buildimage/issues/4999